smalltalksrc/VMMaker/SlangBasicTranslationTest.extension.st

```diff
@@ -11,6 +11,7 @@ SlangBasicTranslationTest >> setUp [
 		labels: Set new;
 		definingClass: self class;
 		selector: #setUp;
+		parseTree: (TStatementListNode new statements: #[])
```

Comment:
`#[]` is a byte array. It works only because it is empty :).

You should either use:
 - `#()`
 - `{}`
 - or `OrderedCollection new`

For the difference between the first two kind of array, you can check this short video: https://www.youtube.com/watch?v=4R1D0PZ3Z3k&list=PL2okA_2qDJ-kCHVcNXdO5wsUZJCY31zwf&index=83